dashboard/test/ui/features/teacher_tools/rubrics/teacher_view_of_rubric.feature

```diff
@@ -47,4 +47,70 @@ Scenario: Teachers can give and send feedback on the rubric to students.
   And I click selector "strong:contains(Code Quality)" once I see it
   And I wait until element "textarea:contains(Nice work Lillian!)" is visible
 
+  @eyes
+  @skip
+  Scenario: Teacher views Rubric and Settings tabs, without accordion experiment
```

Comment:
Minor nit, but the original layout was accordion. The new experiment is carousel.

Reply:
Good catch! That could have been confusing later